Examples/test-suite/overload_simple.i

```diff
@@ -166,10 +166,21 @@ char *fid(double, double) {
 %}
 
 %inline %{
-unsigned long long ull() { return 0ULL; }
-unsigned long long ull(unsigned long long ull) { return ull; }
-long long ll() { return 0LL; }
-long long ll(long long ull) { return ull; }
+#include <errno.h>
+#include <math.h>
+
+int sizeof_long() { return sizeof(long); }
+double getnextafter(double from, double to) { double ret = nextafter(from, to); errno = 0; return ret; }
```

Comment:
`nextafter()` is C99 so this testcase may fail if using a C90 compiler which is still the baseline we aim to support.

I don't think we have a "C99-only testcase" mechanism, but `nextafter()` is also C++11 and we do have a "C++11-only testcase" mechanism so this could be made a C++11 testcase (rename to add a `cpp11_` prefix and move it to `CPP11_TEST_CASES`).  Or just drop this helper and skip the parts of the `_runme.py` that requires it if using Python < 3.9.

Reply:
I've dropped this C helpers and added an optional fallback to numpy instead. If both is missing, the tests will be skipped. I hope the coverage is still good enough.
Our behavior changed with Python >= 3.10, so having at least 3.9 should still test all required cases.